testbed/addons/awslb/construct.ts

```diff
@@ -0,0 +1,47 @@
+import cdk = require('@aws-cdk/core');
+import eks = require('@aws-cdk/aws-eks');
+import iam = require('@aws-cdk/aws-iam');
+import * as yaml from 'js-yaml';
+import * as request from 'sync-request';
+
+export interface AWSLoadBalancerControllerProps {
+    cluster: eks.Cluster;
+    namespace: string;
+}
+
+export class AWSLoadBalancerController extends cdk.Construct {
+    constructor(scope: cdk.Construct, id: string, props: AWSLoadBalancerControllerProps) {
+        super(scope, id);
+
+        const sa = props.cluster.addServiceAccount('aws-lbc', {
+            name: "aws-lbc",
+            namespace: props.namespace
+        });
+
+        sa.role.addManagedPolicy({
+            //to-do  move this to managed policy once it's added to IAM :https://github.com/awslabs/kubernetes-iteration-toolkit/issues/6
+            managedPolicyArn: `arn:aws:iam::aws:policy/AdministratorAccess`
+        });
+
+        const awsLbcCrdsManifestResource = props.cluster.addManifest('awsLbcCrdManifest', ...yaml.loadAll(request.default('GET', 'https://raw.githubusercontent.com/aws/eks-charts/master/stable/aws-load-balancer-controller/crds/crds.yaml').getBody().toString()));
+
+        const chart = props.cluster.addHelmChart('AWSLBCHelmChart', {
+            chart: 'aws-load-balancer-controller',
+            release: 'aws-load-balancer-controller',
+            repository: 'https://aws.github.io/eks-charts',
+            namespace: props.namespace,
+            createNamespace: false,
```

Comment:
Is this the default value?

Reply:
default is `true`

Reply:
Man CDK's implementation decisions drive me crazy sometimes.